### DISCUSSION TITLE
github.com/projectdiscovery/utils/maps modules requires Go 1.19

### DISCUSSION THREAD

**Discussion by ariosrandy** [projectdiscovery/subfinder#777] [Q&A]
fresh installation when i run **go build** (install with git clone) always give me this error message

**# github.com/projectdiscovery/utils/maps
/home/randy/go/pkg/mod/github.com/projectdiscovery/utils@v0.0.10/maps/synclock_map.go:15:18: undefined: atomic.Bool
note: module requires Go 1.19**

the same error also come when install from go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest

@ehsandeep

**Comment by ehsandeep** [comment]
@ariosrandy what version of go you have installed on your system? `go version`

**Comment by ehsandeep** [comment]
also had a typo in the readme which is now updated here - https://github.com/projectdiscovery/subfinder/pull/776

**Comment by ariosrandy** [comment]
> @ariosrandy what version of go you have installed on your system? `go version`

randy@LAPTOP-EPRR1I3R:~$ go version
go version go1.18.1 linux/amd64

@ehsandeep when i try to apt upgrade golang 

golang is already the newest version (2:1.18~0ubuntu2).

**Comment by ehsandeep** [comment]
@ariosrandy [latest version of go](https://go.dev/dl/) is **1.20**, the required version to install subfinder is **1.19**, and you have go installed **1.18** which is outdated.
